readthedocs/templates/projects/integration_webhook_detail.html

```diff
@@ -46,45 +42,47 @@
           You can use the following address to manually configure this webhook.
         {% endblocktrans %}
       </p>
-    {% else %}
+    {% endif %}
+  {% endif %}
+
+  <p>
+    {% blocktrans trimmed %}
+      To manually configure this webhook with your provider, use the
+      following information:
+    {% endblocktrans %}
+  </p>
+
+  <p>
+    {% url 'api_webhook' project_slug=project.slug integration_pk=integration.pk as webhook_url %}
```

Comment:
This could use a header "Webhook URL" or something, similar to the secret below has a header "Secret"